es7-persistence/dependencies.lock

```diff
@@ -14,6 +14,12 @@
         "com.google.guava:guava": {
             "locked": "30.0-jre"
         },
+        "com.google.inject:guice": {
```

Comment:
These additions are not there in Netflix conductor version (https://github.com/Netflix/conductor/blob/main/es7-persistence/dependencies.lock).  Any reason to add these dependencies?  Can we sync with the latest netflix main to get their ES7 changes?

Reply:
As mentioned in the description https://github.com/Netflix/conductor/tree/main/es7-persistence, for using es7 we need to delete all dependencies.lock files and then run:

./gradlew generateLock updateLock saveLock

The line com.google.inject:guice is generated during running above command.